### DISCUSSION TITLE
[Feature] Android app - View albums photo belongs to

### DISCUSSION THREAD

**Discussion by AFumi39** [immich-app/immich#12503] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It's strange because I can't find a similar question/request made before, but, in Android App, it would be very useful to view the list of albums in which an image is, in the "info" of an image.
The same way as the web show it, the app should do the same.
![immich_web_photo_detail](https://github.com/user-attachments/assets/2de00e23-abf5-4404-8d95-4d2946256683)
![Screenshot_2024-09-09-13-38-54-947_app alextran immich](https://github.com/user-attachments/assets/2d7a57f3-da4c-4230-a72b-47f9ad73a85e)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#11615
